inspirehep/modules/theme/templates/inspirehep_theme/invenio_record_editor/index.html

```diff
@@ -0,0 +1,53 @@
+{#
+# This file is part of INSPIRE.
+# Copyright (C) 2016 CERN.
+#
+# INSPIRE is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+#}
+
+{%- extends config.RECORD_EDITOR_BASE_TEMPLATE -%}
+
+{%- block page_body %}
+<div id="re-wrapper">
+	<re-app>
+	Loading...
+	</re-app>
+</div>
+
+{% endblock page_body %}
+
+
+{%- block page_footer %}
+{%- endblock page_footer %}
+ 
+{% block javascript %}
+{{ super() }}
+<script type="text/javascript">
```

Comment:
Since we use `requirejs` on INSPIRE the code below has to follow the pattern like https://github.com/inspirehep/inspire-next/blob/master/inspirehep/modules/tools/templates/authorlist_form.html#L64 requiring jquery, otherwise an error of `$ undefined` is thrown.